### PR TITLE
[SN1-528] [SN1-529] [SN1-535] Fix/llm edge cases

### PR DESCRIPTION
**Pull request by ronnie-samaroo**

### CODE REVIEW

**Review by wendellswa06 — APPROVED** [review]

File: tests/miners/test_llm.py [hunk 1]
```diff
@@ -78,6 +78,20 @@ def test_llm_query_handler(self):
         print("--- Interpreted result ---")
         print(interpreted_result)
 
+    def test_edge_cases(self):
```
Comment (aph5nt) [inline tests/miners/test_llm.py:81]: nice, but we need moooore test queries,
for now we know it works only for "Return x tx from my addrexx xyz"
I had broken this code with other prompts related to my address
TODO:
- break it, with strange blockchain prompts

**Review by aph5nt — CHANGES_REQUESTED** [review]
